tests/requirements.txt

```diff
@@ -4,20 +4,20 @@ certifi==2022.12.7
 charset-normalizer==3.1.0
 exceptiongroup==1.1.1
 execnet==1.9.0
-grpcio==1.51.3
+grpcio>=1.53.0
 grpcio-tools==1.51.3
-idna==3.4
+idna<3
 iniconfig==2.0.0
 MarkupSafe==2.1.2
 packaging==23.0
 pluggy==1.0.0
-protobuf==4.22.1
-PyJWT==2.6.0
+protobuf>=4.22.1
+PyJWT>=2.6.0
 pytest==7.2.2
 pytest-grpc==0.8.0
 pytest-httpserver==1.0.6
-python-ulid==1.1.0
-requests==2.28.2
+python-ulid>=1.1.0
+requests==2.25.1,<2.30.0
```

Comment:
Updating the test requirements to be less strict so hopefully we catch package compatibility issues in the CI in the future.